extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -52,6 +52,9 @@ export class AmpAd extends AMP.BaseElement {
         // Unspecified or empty type.  Nothing to do here except bail out.
         return null;
       }
+      window.ampAdSlotIdCounter = window.ampAdSlotIdCounter || 0;
+      const slotId = window.ampAdSlotIdCounter++;
+      this.element.setAttribute('data-slot-id', slotId);
```

Comment:
If I rename it to "data-amp-slot-id", would that cut down the risk of collision sufficiently?